### PR TITLE
chars: add page

### PR DESCRIPTION
**Pull request by AnimiVulpis**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/chars.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# chars
+
+> Use this tool to display names and codes for various ASCII (and unicode) characters / code points.
```
Comment (mebeim) [inline pages/common/chars.md]:
```suggestion
> Display names and codes for various ASCII and Unicode characters.
```
Impersonal + simplified.

File: pages/common/chars.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+> More information: <https://github.com/antifuchs/chars>.
+
+- Look up a character by its face value:
```
Comment (mebeim) [inline pages/common/chars.md]:
```suggestion
- Look up a character by its value:
```
I don't think the word "face" adds anything to the sentence here. Does it have a special meaning that I'm missing? If not, I'd suggest removing it.

File: pages/common/chars.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`chars '{{ß}}'`
+
+- Look up a character by its unicode point:
```
Comment (mebeim) [inline pages/common/chars.md]:
```suggestion
- Look up a character by its Unicode code point:
```
The correct term would be "Unicode code point".

File: pages/common/chars.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`chars {{U+1F63C}}`
+
+- Look up a character by ambiguous "char code":
```
Comment (mebeim) [inline pages/common/chars.md]:
```suggestion
- Look up possible characters given an ambiguous code point:
```
I think this reads clearer.

File: pages/common/chars.md (hunk 2 above)
Comment (AnimiVulpis) [reply pages/common/chars.md]: It's just stressing the fact that you can enter the value into the program as you **see** it.

I will remove the "face" part

**Pull request merged**